test/nuxeo-tooltip-test.html

```diff
@@ -0,0 +1,85 @@
+<!--
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<html>
+<head>
+  <title>nuxeo-tooltip tests</title>
+
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
+  <script>
+    // force use of polyfill: see ELEMENTS-56
+    window.fetch = null;
+  </script>
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+  <script src="../../web-component-tester/browser.js"></script>
+  <script src="../../nuxeo-elements/test/test-helpers.js"></script>
+  <link rel="import" href="../../polymer/polymer.html">
+  <link rel="import" href="../widgets/nuxeo-tooltip.html">gi
+</head>
+<body>
+
+  <test-fixture id="hidden">
+    <template>
+      <nuxeo-tooltip hidden>Hello</nuxeo-tooltip>
+    </template>
+  </test-fixture>
+
+  <test-fixture id="notHidden">
+    <template>
+      <nuxeo-tooltip>Hello</nuxeo-tooltip>
+    </template>
+  </test-fixture>
+
+
+  <script>
+    /* eslint-disable no-unused-expressions */
+    function isElementVisible(el) {
+      if (!el) {
+        return false;
+      }
+      const styles = window.getComputedStyle(el);
+      return el.offsetWidth !== 0 && el.offsetHeight !== 0 && styles.opacity > 0 && styles.visibility !== 'hidden';
+    }
```

Comment:
It may be worth to move it [here](https://github.com/nuxeo/nuxeo-elements/blob/maintenance-2.4.x/test/test-helpers.js), I guess.
Low risk and it would be available for every suite (as on master), since it is generic and useful. 🙂

Reply:
thanks! I wanted to do it but was lost in respect to the repo/branches :)

Reply:
pr here: https://github.com/nuxeo/nuxeo-elements/pull/238